### DISCUSSION TITLE
How can I make the serverReadyAction configuration in the .vscode/launch.json work?

### DISCUSSION THREAD

**Discussion by gfranciscoerazom** [fastapi/fastapi#13082] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "My Python Debugger: FastAPI",
            "type": "debugpy",
            "request": "launch",
            "module": "fastapi",
            "args": [
                "dev",
                "./api/main.py"
            ],
            "jinja": true,
            "serverReadyAction": {
                "pattern": ".+Server started at http://127.0.0.1:([0-9]+).*",
                "uriFormat": "http://localhost:%s",
                "action": "openExternally",
            }
        }
    ]
}
```

### Description

I am trying to configure my .vscode/launch.json file for my FastAPI project. Because my understanding is that by configuring the serverReadyAction function, I can make VSCode open a browser with the specified path based on a regex pattern.
The issue is that it is not opening the browser for me, but the debugger is working. And I have tried several patterns, but none of them work.
Reading a bit, it seems that the serverReadyAction > pattern looks for the pattern in the debug console and when I run the debug the logs are not shown in the debug console, but they are shown in the terminal. So maybe that's where the problem is, but I don't know how to solve it. Although I wouldn't be sure if that's the problem either.

### Operating System

Windows

### FastAPI Version

0.115.6

### Pydantic Version

2.10.3

### Python Version

Python 3.11.9

### Additional Context

[VSCode docs](https://code.visualstudio.com/docs/editor/debugging#_automatically-open-a-uri-when-debugging-a-server-program)

**Comment by YuriiMotov** [comment]
Let's close this for now to keep the discussions clean.
If you’ve found a solution, please share it here.
If you still need help, feel free to let us know.
